charts/gitops-server/values.yaml

```diff
@@ -108,3 +108,14 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+serverTLS:
+  # This block is meant to use a secret as a source for the tls info
```

Comment:
I think we should only offer one method for providing certs and it should probably be by creating a secret.